flowauth/backend/flowauth/admin.py

```diff
@@ -868,3 +866,25 @@ def list_all_tokens():
             for token in Token.query.all()
         ]
     )
+
+
+@blueprint.route("/public_key")
+@login_required
+@admin_permission.require(http_exception=401)
+def get_public_key():
+    """
+    Get the public key which can be used to verify tokens for this
+    flowauth server.
+    """
+    key = (
+        current_app.config["PRIVATE_JWT_SIGNING_KEY"]
```

Comment:
Not a big deal, but it's slightly non-intuitive that you need to load the private key from the config in order to return the public one. Would it make sense to define `current_app.config["PUBLIC_JWT_SIGNING_KEY"]` in the config so that we can retrieve it directly where needed?

Reply:
I had it set up like that originally, but it seemed weird to have a config item which was a. only used in one place, and b. was a function of another config item.

Reply:
Personally, I find readability and clarity sufficient as a reason to have a separate definition even if it's only used in a single place. (After all, it's also useful to wrap up code in a function if it hides unnecessary details even if that function is only called in a single place.) But not a big deal in this case, I'll leave it to you what you prefer.